.env

```diff
@@ -6,5 +6,18 @@ REACT_APP_PRIVATENODE='https://node.snowapi.net/'
 REACT_APP_IPFS_API_URL='https://ipfs.snowapi.net/ipfs/'
 REACT_APP_APIADDR='https://api.snowapi.net/graphql'
 REACT_APP_DEVAPIADDR='https://api.snowapi.net/graphql'
+# analytics
 REACT_APP_ANALYTICS_ENDPOINT='//analytics.snowapi.net/'
-REACT_APP_ANALYTICS_SITE_ID=3
\ No newline at end of file
+REACT_APP_ANALYTICS_SITE_ID=3
+# Governance
+REACT_APP_GOVERNANCE_TOKEN_NAME='xSNOB'
+REACT_APP_GOVERNANCE_TOKEN_LOGO_ADDRESS='https://app.snowball.network/_next/image?url=%2Fassets%2Fimages%2Flogo.svg&w=64&q=75'
+REACT_APP_GOVERNANCE_TOKEN_CONTRACT_ADDRESS='0x83952E7ab4aca74ca96217D6F8f7591BEaD6D64E'
+REACT_APP_GOVERNANCE_INFO_LINK='https://docs.snowball.network/governance/community-proposals'
+REACT_APP_MINIMUM_TOKEN_FOR_VOTING='100000'
+REACT_APP_MINIMUM_VOTING_PERIOD='3'
+REACT_APP_MAXIMUM_VOTING_PERIOD='30'
+REACT_APP_MINIMUM_VOTING_PERIOD_UNIT='days'
+#gaugeProxy
+REACT_APP_GAUGE_PROXY_ADDRESS='0x215D5eDEb6A6a3f84AE9d72962FEaCCdF815BF27'
+REACT_APP_GAUGE_PROXY_LAST_INFO_QUERY="{LastSnowballInfo{  createdAt  snowballTVL  blocksPast24hrs  snowballTVL  snobPerBlock  blockHeight  snobNextPhase  snowballToken {    supply    totalSupply    pangolinPrice  }  poolsInfo{    address    lpAddress    name    kind    source    symbol    tvlStaked    dailyAPR    dailyAPY    weeklyAPY    yearlyAPY    pricePoolToken    deprecated    token0{      address      name      symbol      pangolinPrice      supply      decimals    }    token1{      address      name      symbol      pangolinPrice      supply      decimals    }    token2{      address      name      symbol      pangolinPrice      supply      decimals    }    token3{      address      name      symbol      pangolinPrice      supply      decimals    }    gaugeInfo{      address      tvlStaked      snobDailyAPR      snobWeeklyAPR      snobYearlyAPR      fullDailyAPY      fullWeeklyAPY      fullYearlyAPY      snobAllocation    }  }}}"
```

Comment:
there should be a folder on the services where we can define these queries. These arent env vars. Please move them where the apis live.